policyengine_us/variables/gov/states/me/tax/income/taxable_income/me_deduction.py

```diff
@@ -0,0 +1,11 @@
+from policyengine_us.model_api import *
+
+
+class me_deduction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Maine deduction"
```

Comment:
should this be plural?

Reply:
That was my first thought but I deferred to the existing one for NY. I think it's because it'll be calculated as max(itemized,standard) so after that it's just one?

Reply:
Ah, I think we should use `deductions` and change the NY one.

We do that for federal: https://github.com/PolicyEngine/policyengine-us/blob/master/policyengine_us/variables/gov/irs/income/taxable_income/deductions/taxable_income_deductions.py